README.md

```diff
@@ -1,24 +1,114 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Rent-A-Green
```

Comment:
- [ ] It would be great if you post a screenshot of your project.  Since this is an API, the swagger page would be great!